acceptance/script.prepare

```diff
@@ -34,3 +31,12 @@ trace() {
 
     return $?
 }
+
+git-repo-init() {
+    git init -qb main
+    git config --global core.autocrlf false
```

Comment:
@pietern  Found out about this option because a warning was printed by git about this and was captured in test output. This must be the reason we need to fix crlf in golden files in  integration tests.

Reply:
It can be set in `.gitattributes`: https://docs.github.com/en/get-started/getting-started-with-git/configuring-git-to-handle-line-endings#per-repository-settings